packages/nuxt/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@hey-api/nuxt",
+  "version": "0.0.0",
+  "description": "🚀 Nuxt module for `@hey-api/openapi-ts` codegen.",
+  "homepage": "https://heyapi.dev/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hey-api/openapi-ts.git"
+  },
+  "bugs": {
+    "url": "https://github.com/hey-api/openapi-ts/issues"
+  },
+  "license": "MIT",
+  "author": {
+    "email": "lubos@heyapi.dev",
+    "name": "Hey API",
+    "url": "https://heyapi.dev"
+  },
+  "funding": "https://github.com/sponsors/hey-api",
+  "keywords": [
+    "client",
+    "codegen",
+    "fetch",
+    "http",
+    "javascript",
+    "nuxt",
+    "openapi",
+    "rest",
+    "swagger",
+    "typescript",
+    "vue"
+  ],
+  "type": "module",
+  "main": "./dist/module.js",
+  "exports": {
+    ".": "./dist/module.js",
+    "./module": "./dist/module.js",
+    "./package.json": "./package.json"
+  },
+  "typesVersions": {
+    "*": {
+      ".": [
+        "./dist/module.d.ts"
+      ],
+      "module": [
+        "./dist/module.d.ts"
+      ]
+    }
+  },
+  "sideEffects": false,
+  "files": [
+    "dist",
+    "LICENSE.md",
+    "src"
+  ],
+  "scripts": {
+    "build": "tsup && rollup -c && pnpm check-exports",
```

Comment:
I'd recommend you use nuxt/module-builder

you can see an example at https://github.com/nuxt/fonts

Reply:
Oh that looks neat, will try, thank you!